doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2225,17 +2284,17 @@ nRF5340 Audio
 
```

Comment:
Please add:
OCT-3368: For 16 and 24 kHz: The application may repeatedly print: "audio_sync_timer: Unable to get new CC value" which may cause degraded audio quality.

.. rst-class:: v3-0-0